lib/features/family/features/reflect/domain/reflect_and_share_repository.dart

```diff
@@ -19,6 +20,11 @@ class ReflectAndShareRepository {
 
   final List<String> _usedSecretWords = [];
 
+  void saveGratitudeInterestsForCurrentSuperhero(List<Tag>? interests) {
+    _selectedProfiles[_getCurrentSuperHeroIndex()] =
```

Comment:
Somewhere also a translation need to happen between the 6 categories and the tag we know from the Charity Finder. Is that going to be this place? Or the cubit thats gonna call this method?

Reply:
Not this place, the cubit that calls it. Wether its 6/12 or whatever amount of categories, at the end of the day I think it still just a list of interests, which is all we need for the domain.

Reply:
I have the catgories containing the tags hardcoded, maybe we take care of this later